src/main/scala/pipelines/images/cifar/RandomPatchCifarAugmented.scala

```diff
@@ -99,7 +99,7 @@ object RandomPatchCifarAugmented extends Serializable with Logging {
     val testPredictions = predictionPipeline(testImagesAugmented)
 
     val testEval = AugmentedExamplesEvaluator(
-      testImageIdsAugmented, testPredictions, testLabelsAugmented, numClasses)
+      testImageIdsAugmented, testPredictions.get, testLabelsAugmented, numClasses)
```

Comment:
We actually only have an [RDD,RDD] interface right now. testLabelsAugmented is an RDD in the current pipeline. For the most part our labels end up being RDD while the predictions end up being `PipelineDataset` which is why I have the [PipelineDataset,RDD] interface in the multiclass classifier evaluator. I agree there should be consistency here, though. Maybe there's something we can do at the level of an "evaluator interface" that will obviate the need for duplicating a bunch of boilerplate logic.